kedro/framework/cli/utils.py

```diff
@@ -460,37 +438,21 @@ def _split_params(ctx, param, value):
 
 
 def _split_load_versions(ctx, param, value):
-    lv_tuple = _get_values_as_tuple([value])
-    return _reformat_load_versions(ctx, param, lv_tuple) if value else {}
-
-
-def _get_values_as_tuple(values: Iterable[str]) -> tuple[str, ...]:
-    return tuple(chain.from_iterable(value.split(",") for value in values))
-
-
-def _deprecate_options(ctx, param, value):
-    deprecated_flag = {
-        "node_names": "--node",
-        "tag": "--tag",
-        "load_version": "--load-version",
-    }
-    new_flag = {
-        "node_names": "--nodes",
-        "tag": "--tags",
-        "load_version": "--load-versions",
-    }
-    shorthand_flag = {
-        "node_names": "-n",
-        "tag": "-t",
-        "load_version": "-lv",
-    }
-    if value:
-        deprecation_message = (
-            f"DeprecationWarning: 'kedro run' flag '{deprecated_flag[param.name]}' is deprecated "
-            "and will not be available from Kedro 0.19.0. "
-            f"Use the flag '{new_flag[param.name]}' instead. Shorthand "
-            f"'{shorthand_flag[param.name]}' will be updated to use "
-            f"'{new_flag[param.name]}' in Kedro 0.19.0."
-        )
-        click.secho(deprecation_message, fg="red")
-    return value
+    if not value:
+        return {}
+
+    lv_tuple = tuple(chain.from_iterable(value.split(",") for value in [value]))
+
+    load_versions_dict = {}
+    for load_version in lv_tuple:
+        load_version = load_version.strip()  # noqa: PLW2901
+        load_version_list = load_version.split(":", 1)
+        if len(load_version_list) != 2:  # noqa: PLR2004
+            raise KedroCliError(
+                f"Expected the form of 'load_version' to be "
```

Comment:
This should be plural `load_versions` now right?
```suggestion
                f"Expected the form of 'load_versions' to be "
```